lib/gui/elistboxcontent.cpp

```diff
@@ -959,11 +959,16 @@ int eListboxPythonMultiContent::getMaxItemTextWidth()
 				{
 				case TYPE_TEXT: // text
 				{
-					ePyObject px = PyTuple_GET_ITEM(item, 1), pstring = PyTuple_GET_ITEM(item, 7);
+					ePyObject px = PyTuple_GET_ITEM(item, 1), pfnt = PyTuple_GET_ITEM(item, 5), pstring = PyTuple_GET_ITEM(item, 7);
 
 					if (PyLong_Check(pstring) && data) /* if the string is in fact a number, it refers to the 'data' list. */
 						pstring = PyTuple_GetItem(data, PyLong_AsLong(pstring));
 
+					if (pfnt) {
```

Comment:
I think there should be an additional check if the following font is in the list:
`if (pfnt) && !(m_font.find(fnt) == m_font.end()) {`
https://github.com/OpenViX/enigma2/blob/Developer/lib/gui/elistboxcontent.cpp#L1168-L1172